manager/assets/modext/widgets/media/modx.browser.js

```diff
@@ -275,8 +275,9 @@ Ext.extend(MODx.browser.View,MODx.DataView,{
         var node = this.getSelectedNodes();
         var detailPanel = Ext.getCmp(this.config.ident+'-img-detail-panel').body;
         var okBtn = Ext.getCmp(this.ident+'-ok-btn');
+		var keys = Object.keys(node);
```

Comment:
Just to note this, `Object.keys` is only available as of IE9. According to the [docs](https://docs.modx.com/revolution/2.x/getting-started/server-requirements), the expected browser support goes back to IE8 which [is apparently down to 0.14% yay](https://caniuse.com/usage-table). 

Perhaps 3.0 is a good time to bump the minimum to, say, IE11? :)

Reply:
Since this PR is marked as ready to be merged; are we going to bump the minimum IE version MODX supports in 3.x?  Maybe we should drop support for IE in 3.x